src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4encoder/common/AudioControllingMetric.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4encoder.common;
+
+/**
+ * AudioControllingMetric class defined the enum for the monitoring process
+ *
+ * @author Ivan
+ * @version 1.0
+ * @since 1.0
```

Comment:
Yes, I added it. This is my configuration file on Intellij:

`#if (${PACKAGE_NAME} && ${PACKAGE_NAME} != "")package ${PACKAGE_NAME};#end
#parse("File Header.java")
/**
 * ${NAME} 
 * 
 * @author Ivan / Symphony Dev Team<br>
 *		   Created on ${DATE}
 * @since 1.0.0
 */
public class ${NAME} {
}`